lib/registry.js

```diff
@@ -236,7 +236,7 @@ Registry.prototype.configureModel = function(ModelCtor, config) {
     ModelCtor.attachTo(config.dataSource);
     debug('Attached model `%s` to dataSource `%s`',
       modelName, config.dataSource.name);
-  } else if (config.dataSource === null) {
+  } else if (config.dataSource === null || config.dataSource === false) {
```

Comment:
Above doesn't handle `undefined`, is that intentional?

Why not `if (!config.dataSource) {`, so that all falsy values are treated equally?